app/views/units/create.js.erb

```diff
@@ -0,0 +1 @@
+$("#contents").append("<li id=\"unit-<%= @unit.id %>\"> <%=@unit.name%> <a data-remote=\"true\" rel=\"nofollow\" data-method=\"delete\" href=\"/units/<%=@unit.id%>\">Delete Unit</a>");
```

Comment:
Can you tell me more about the pros and cons of the different approaches? I learned this way from the Rails guides and it seems really easy so I'm a little daunted at the prospect of switching without knowing why.